### ISSUE TITLE
[BUG] sidestore gets stuck refreshing forever

### ISSUE THREAD

**Issue by thriftydoge** [SideStore/SideStore#302]
### Describe the bug

Sidestore gets stuck attempting to refresh apps. Nothing appears in minimuxer logs besides several "Heartbeat success!" messages. I couldn't check the final errors because the app crashed after approximately 26 minutes of waiting. 

### Instructions to reproduce

Open sidestore

Navigate to "My Apps"

Press refresh all

### What version of SideStore are you using?

0.3.0-m-f1shy-mdc.14

### Other info

A video of me trying to refresh my apps (set to 40X speed for your viewing pleasure)

https://user-images.githubusercontent.com/87211537/225104563-d70b1042-e2d4-456d-8c80-7f96387bb238.mov

**Comment by lingsoul** [comment]
Same happens with me
Version: 0.3.2-nightly.2023.03.21.1
![image](https://user-images.githubusercontent.com/22099083/227085018-4dcbde68-4b38-4d81-aaa5-e0f920776435.png)

**Comment by SHAD0WDEM0N** [comment]
I am having the Same Problem Here, Im new to Sidestore and thought i had done something wrong until i see use have the exact same issue, I cant even sideload an IPA

**Comment by jiangyiUI** [comment]
Same Problem

**Comment by SHAD0WDEM0N** [comment]
HERE IS THE FIX
I looked on the discord and i got a fix its a different IPA with the fixes for multiple bugs i will attach below

(Had to create zip due to GitHub support Download and there is the IPA i will also link the discord i received this)

Discord Invite: https://discord.gg/RgpFBX3Q3k
Message: https://discord.com/channels/949183273383395328/1037581733144559676/1088318838694756452
File: [Sidestore Fix.zip](https://github.com/SideStore/SideStore/files/11060913/Sidestore.Fix.zip)

**Comment by naturecodevoid** [comment]
there's an updated build of that in announcements

**Comment by menotuu** [comment]
For me:
Build sidestore 0.3.2 nightly 2023.03.21.1: Loading forever
Logs:
Minimuxer many times per second:


```
22:10:37 [INFO] Logger initialized!!
22:10:37 [INFO] minimuxer has started!
22:10:37 [INFO] Starting image mounter
22:10:37 [INFO] Starting heartbeat thread
22:10:37 [WARN] Could not get device from muxer for heartbeat: NoDevice

```

sidestore: https://paste.kodi.tv/jehadaxuku.kodi

**Comment by simonez17** [comment]
> HERE IS THE FIX I looked on the discord and i got a fix its a different IPA with the fixes for multiple bugs i will attach below
> 
> (Had to create zip due to GitHub support Download and there is the IPA i will also link the discord i received this)
> 
> Discord Invite: https://discord.gg/RgpFBX3Q3k Message: https://discord.com/channels/949183273383395328/1037581733144559676/1088318838694756452 File: [Sidestore Fix.zip](https://github.com/SideStore/SideStore/files/11060913/Sidestore.Fix.zip)

This version have provisioning fix that is on the nigthly and fix the bug of 0.3.2-nightly.2023.03.21.1 of stuck refresh, thank you so much!!
